### PR TITLE
Explain helm chart install on restricted environments

### PR DESCRIPTION
**Pull request by josvazg**
**Description of the change**

Users within an enterprise might face very restrictive installation environments for Sealed Secrets.

This change adds documentation specific on how to install from the Helm Chart on one of the most restrictive cases out there:
- Access to a single `namespace` and `service account` only, no cluster access at all.

**Benefits**

Users facing restrictive Kubernetes environment have now easy to follow instructions to install from the chart.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,6 +304,49 @@ Alternatively, you can set `fullnameOverride` when installing the chart to overr
...
+  limits: 
+    cpu: 150m
+    memory: 256Mi
+```
+
+Note that:
+- No service accounts are created, instead the one allocated to you will be used.
+  - `{allocated-service-account}` is the name of the `service account` you were allocated on the cluster.
+- No RBAC roles are created neither in the namespace or the cluster.
+- Resource limits must be espeficied.
```
Comment (tewfik-ghariani) [inline README.md:339]: Typo ^^

File: README.md [hunk 2]
```diff
@@ -304,6 +304,49 @@ Alternatively, you can set `fullnameOverride` when installing the chart to overr
...
+  - The limits are samples that shoudl work, but you might want to review them in your parituclar setup.
```
Comment (tewfik-ghariani) [inline README.md:340]: should* and particular*

File: README.md [hunk 3]
```diff
@@ -304,6 +304,49 @@ Alternatively, you can set `fullnameOverride` when installing the chart to overr
...
+resources: 
+  limits: 
+    cpu: 150m
+    memory: 256Mi
+```
+
+Note that:
+- No service accounts are created, instead the one allocated to you will be used.
+  - `{allocated-service-account}` is the name of the `service account` you were allocated on the cluster.
+- No RBAC roles are created neither in the namespace or the cluster.
```
Comment (tewfik-ghariani) [inline README.md:338]: nor* ^^

File: README.md (hunk 1 above)
Comment (josvazg) [reply README.md:339]: Thanks!

**Review by alvneiayu — APPROVED** [review]
LGTM

**Pull request merged**